AudioKit/Common/Nodes/Generators/Oscillators/Oscillator/AKOscillatorDSP.mm

```diff
@@ -6,6 +6,13 @@
 
 #include "DebugDSP.h"
 
+enum AKOscilatorParameter : AUParameterAddress {
```

Comment:
I don't understand how this worked with the typo misspelling of oscillator.  Almost as if nothing refereed to this?

Reply:
It's true, nothing refers to it. The enum values are stringified by the AK_REGISTER_PARAMETER macro.